app/components/bali/card/index.scss

```
@@ -0,0 +1,18 @@
+.column:not(.is-full) {
```

Acostumbramos que el css este dentro de la clase nombre-del-componente-component, para que solo aplique los estilos al componente. Entonces en este caso sería 

```suggestion
.card-component {
  ## CSS del componente
}
```

---

Y sí está bien que lo ponga en esa línea aunque arriba tenga ".column:not(.is-full)"?